garage/contrib/ros/docker/sawyer/docker-entrypoint.sh

```diff
@@ -0,0 +1,253 @@
+#!/bin/bash
+
+ORANGE='\033[0;33m'
+WHITE='\033[1;37m'
+
+export USER=sawyer_docker
+export GARAGE_PYTHON=/opt/conda/envs/garage/bin/python
+
+# Clean up bashrc
+echo "export USER=sawyer_docker" > "/home/$USER/.bashrc"
+
+echo "source /opt/ros/kinetic/setup.bash" >> "/home/$USER/.bashrc"
+source /opt/ros/kinetic/setup.bash
+
+echo "export USER=$USER" >> "/home/$USER/.bashrc"
+echo "export HOME=/home/$USER" >> "/home/$USER/.bashrc"
+
+export PATH=$PATH:/opt/conda/bin
+echo 'export PATH=$PATH:/opt/conda/bin' >> "/home/$USER/.bashrc"
```

Comment:
Please don't produce unnecessary prints to the console. Unless the user absolutely needs to know something for the behavior of the command to be meaningful, remain silent.